### PR TITLE
+doc security page about recent vunerability

### PR DESCRIPTION
**Pull request by ktoso**
Please review @patriknw @jrudolph

### CODE REVIEW

File: docs/src/main/paradox/security.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Receiving security advisories
+The best way to receive any and all security announcements is to subscribe to the [Akka security list](https://groups.google.com/forum/#!forum/akka-security).
+
+The mailing list is very low traffic, and receives notifications only after Security reports have been managed by the core team and fixes are publicly available.
+
+## Reporting vulnerabilities
+We strongly encourage people to report such problems to our private security mailing list first, before disclosing them in a public forum.
+
+All security bugs in Play should be reported by email to security@lightbend.com.
```
Comment (jrudolph) [inline docs/src/main/paradox/security.md]: "Play"? ;)

WDYT of this wording for this section:

```
Following best-practice, we strongly encourage anyone to report potential security vulnerabilities to security@lightbend.com before disclosing them in a public forum like the mailing list or as a Github issue.

Reports to this email address will be handled by our security experts with priority. We'll work together with you
to ensure that a fix can be provided without delay.
```

File: docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+All directories and files on the same drive as the specified directory for which the server process has 
+sufficient permissions may be downloaded or browsed. This can be easily exploited by using a specially 
+crafted URI. 
+
+For example, such specially crafted request `http://localhost:8080/%5c..%5c..%5c..%5c..%5c..%5c..%5c..%5c..%5c..%5cwindows/win.ini` 
+when handles by one of the affected directives, could expose your win.ini (and potentially any other file) to the attacker.
+
+## Impact
+
+Directory traversal in case the above vunerable directives are used.
```
Comment (jrudolph) [inline docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md]: "vulnerable"

File: docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+## Recommendations
+
+Following best security practices it is furthermore recommended to run the web server 
+process with user credentials with as few permissions as possible to prevent unintended file access.  
+
+Furthermore, we suggest using Linux servers and/or containers for hosting Akka HTTP applications, 
+as these OSes receive more scrutiny than any other OS just because of the overwhelming number of 
+installations running on Linux.
+
+Following best security practices it is furthermore recommended to run the web server process with
```
Comment (jrudolph) [inline docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md]: 2 duplicated paragraphs below

File: docs/src/main/paradox/security.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Security annoucements
```
Comment (ktoso) [inline docs/src/main/paradox/security.md]: thanks

**Comment by ktoso** [comment]
Hey @wsargent, could you please review this? :)

File: docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md (hunk 3 above)
Comment (ktoso) [inline docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md]: whoops, fixed

File: docs/src/main/paradox/security.md (hunk 1 above)
Comment (ktoso) [inline docs/src/main/paradox/security.md]: ok, 👍

**Comment by ktoso** [comment]
PLS BUILD

**Comment by wsargent** [comment]
@ktoso bit busy today but I'll schedule it

File: docs/src/main/paradox/security.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+Reports to this email address will be handled by our security experts with priority. We'll work together with you
+to ensure that a fix can be provided without delay.
+
+## Fixed Security Vulnerabilities
+
+### Fixed in Akka HTTP 2.4.11
+
+@@@ index
+
+- [Directory Traversal Vulnerability Announcement](security/2016-09-30-windows-directory-traversal.md)
```
Comment (wsargent) [inline docs/src/main/paradox/security.md]: Will internal .md links work?  I've had to quote them in `[[internal]]` link style to work when used outside of the table of contents.

File: docs/src/main/paradox/security.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+The mailing list is very low traffic, and receives notifications only after security reports have been managed by the core team and fixes are publicly available.
+
+## Reporting vulnerabilities
+
+We strongly encourage people to report such problems to our private security mailing list first, before disclosing them in a public forum.
+
+Following best-practice, we strongly encourage anyone to report potential security 
+vulnerabilities to security@lightbend.com before disclosing them in a public forum like the mailing list or as a Github issue.
+
+Reports to this email address will be handled by our security experts with priority. We'll work together with you
```
Comment (wsargent) [inline docs/src/main/paradox/security.md]: "Reports to this email address will be handled by our security team, who will work together with you"

File: docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+On Windows akka-http’s `getFromDirectory`, `getFromBrowseableDirectory`, `getFromBrowseableDirectories`, 
+and `listDirectoryContents` directives unintentionally allow access to directories and files outside of 
+the specified directory. 
+
+All directories and files on the same drive as the specified directory for which the server process has 
+sufficient permissions may be downloaded or browsed. This can be easily exploited by using a specially 
+crafted URI. 
+
+For example, such specially crafted request `http://localhost:8080/%5c..%5c..%5c..%5c..%5c..%5c..%5c..%5c..%5c..%5cwindows/win.ini` 
+when handles by one of the affected directives, could expose your win.ini (and potentially any other file) to the attacker.
```
Comment (wsargent) [inline docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md]: s/handles/handled/

File: docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+On Windows akka-http’s `getFromDirectory`, `getFromBrowseableDirectory`, `getFromBrowseableDirectories`, 
+and `listDirectoryContents` directives unintentionally allow access to directories and files outside of 
+the specified directory. 
+
+All directories and files on the same drive as the specified directory for which the server process has 
+sufficient permissions may be downloaded or browsed. This can be easily exploited by using a specially 
+crafted URI. 
+
+For example, such specially crafted request `http://localhost:8080/%5c..%5c..%5c..%5c..%5c..%5c..%5c..%5c..%5c..%5cwindows/win.ini`
```
Comment (wsargent) [inline docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md]: s/such/a/

File: docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+## Impact
+
+Directory traversal in case the above vulnerable directives are used.
+
+## Affected Versions
+
+OS: 
+
+- **Only Windows** operating systems are affected by this vunerability.
```
Comment (wsargent) [inline docs/src/main/paradox/security/2016-09-30-windows-directory-traversal.md]: s/vunerability/vulnerability/

File: docs/src/main/paradox/security.md (hunk 5 above)
Comment (ktoso) [inline docs/src/main/paradox/security.md]: you're right, I wonder why I thought it's ok - was checking rendered output. Fixed

**Comment by ktoso** [comment]
Thanks for the review @wsargent, amended.

**Pull request merged**